python/ray/serve/application.py

```diff
@@ -0,0 +1,376 @@
+from typing import List, Dict, TextIO, Optional, Union
+import json
+import yaml
+import time
+import sys
+
+import ray
+from ray import serve
+from ray.serve.api import (
+    Deployment,
+    _get_global_client,
+)
+from ray.serve.common import DeploymentStatus
+from ray.dashboard.modules.serve.schema import (
+    ServeApplicationSchema,
+    serve_application_to_schema,
+    schema_to_serve_application,
+    serve_application_status_to_schema,
+)
```

Comment:
is there a way not to have serve depends on dashboard modules?

Reply:
e.g. move the schema to serve instead

Reply:
yeah I think this is a good idea given the structure now

Reply:
I moved it to the serve directory.